### ISSUE TITLE
[EI 6.1] Please specify the correct location when Installing Features Using POM Files

### ISSUE THREAD

**Issue by ushanib** [wso2/product-ei#492]
When trying to install HL7, I referred this doc [1]
[1] https://docs.wso2.com/display/EI610/Installing+Features+Using+POM+Files

It mentions, 
`This example provides a sample pom.xml file to create a new feature installed distribution, which uses the default profile (<profile>default</profile>) and the corresponding directory path (dir="<PRODUCT_NAME>/repository/components/default/configuration/org.eclipse.equinox.simpleconfigurator">). For more information on the plugins, see maven-antrun-plugin and Carbon P2 plugin.`

It should be,

`<EI 6.1_HOME>/wso2/components/default/configuration/org.eclipse.equinox.simpleconfigurator>`

Please mention the correct place in the doc where I should create the relevant pom.xml

**Comment by ushanib** [comment]
Also update the pom.xml as it works for EI 6.1

    ```
   <destination>wso2ei-6.0.0/wso2/components</destination>
       
should be        <destination>wso2ei-6.1.0/wso2/components</destination>
       
```

**Comment by praneesha** [comment]
Fixed the above in EI 6.0.0 & 6.1.0 doc accordingly.
